pdns/tcpreceiver.cc

```diff
@@ -197,10 +197,10 @@ static bool maxConnectionDurationReached(unsigned int maxConnectionDuration, tim
 {
   if (maxConnectionDuration) {
     time_t elapsed = time(nullptr) - start;
-    if (elapsed >= maxConnectionDuration) {
+    if (elapsed > 0 && elapsed >= maxConnectionDuration) {
```

Comment:
why the `elapsed > 0` test here?

Reply:
My initial intent was to avoid treating a "clock has gone backward" event as a timeout, but it does not look necessary. I'll remove it.